### PR TITLE
feat: add docs on integrating with Netlify and Vercel

### PR DESCRIPTION
**Pull request by pgautier404**

### CODE REVIEW

File: docs/develop/integrations/vercel-integration.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+In this tutorial, we'll make use of a simple example [Next.js chat application](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/web/nextjs-chat) from our [Javascript SDK](https://github.com/momentohq/client-sdk-javascript). By the end of the exercise, you'll have a realtime browser-based chat application up and running at a URL you can use to invite your friends to the chat.
+
+## Prerequisites
+
+To get this app deployed and running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- A [Vercel](https://vercel.com/) account.
+- A copy or fork of the [Momento Javascript SDK](https://github.com/momentohq/client-sdk-javascript) in your personal repositories.
+- A Momento authentication token.
```
Comment (NoSQLKnowHow) [inline docs/develop/integrations/vercel-integration.md]: What kind of token does this need to be? A super-user token? If so, this needs to be a link to our console to create that token.

File: docs/develop/integrations/vercel-integration.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- A Momento cache.
```
Comment (NoSQLKnowHow) [inline docs/develop/integrations/vercel-integration.md]: Is this created in the console or programmatically? If console, this should probably be stated and a link to the console provided.

File: docs/develop/integrations/vercel-integration.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Once you have a Vercel account and a copy or fork of the Momento Javascript SDK in your Git provider account, you're ready to configure the Momento side of things using the [Momento console](https://console.gomomento.com). You can create an account in the console by providing your email address or linking an existing Google or GitHub account. Once you've logged into the console, click the `Create Cache` button on the top right of the page:
```
Comment (NoSQLKnowHow) [inline docs/develop/integrations/vercel-integration.md:21]: Wait. You say a token and cache are pre-reqs, but then have procedures here on how to create them. Then they are not pre-reqs, right?

File: docs/develop/integrations/vercel-integration.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+![Vercel root directory](/img/vercel-root-directory.png)
+
+Notice the Next.js icon next to the selected directory. Vercel has examined the contents of the directory and detected that our project is using Next.js. Because of this auto-detection, we can simply leave the `Build and Output Settings` section alone. It will select default values appropriate to our project. 
+
+Recall that we generated and saved a Momento auth token earlier for use with this application. Use the `Environment Variables` section below the build settings to add an environment variable with `MOMENTO_AUTH_TOKEN` as the name and the token you generated as the value. Next, add a second environment variable with `NEXT_PUBLIC_MOMENTO_CACHE_NAME` as the key and `chat` as the value to point your app to the cache you created earlier.
```
Comment (NoSQLKnowHow) [inline docs/develop/integrations/vercel-integration.md:61]: Is there no secure storage for things in Vercel that are kind of like AWS Secrets Manager? If it is possible to have examples using environment variables and using a secrets manager, that'd be best practice.

File: docs/develop/integrations/netlify-integration.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+In this tutorial, we'll make use of a simple example [Next.js chat application](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/web/nextjs-chat) from our [Javascript SDK](https://github.com/momentohq/client-sdk-javascript). By the end of the exercise, you'll have a realtime browser-based chat application up and running at a URL you can use to invite your friends to the chat.
+
+## Prerequisites
+
+To get this app deployed and running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- A [Netlify](https://www.netlify.com/) account.
+- A copy or fork of the [Momento Javascript SDK](https://github.com/momentohq/client-sdk-javascript) in your personal repositories.
+- A Momento authentication token.
```
Comment (NoSQLKnowHow) [inline docs/develop/integrations/netlify-integration.md]: is this a pre-req if you show someone how to do it below? IMO, it needs to be one or the other. thoughts?

File: docs/develop/integrations/netlify-integration.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- A Momento cache.
```
Comment (cprice404) [inline docs/develop/integrations/netlify-integration.md]:
```suggestion
```

File: docs/develop/integrations/vercel-integration.md (hunk 2 above)
Comment (cprice404) [inline docs/develop/integrations/vercel-integration.md]:
```suggestion
```

File: docs/develop/integrations/vercel-integration.md (hunk 4 above)
Comment (cprice404) [reply docs/develop/integrations/vercel-integration.md:61]: this is a good call-out but I think we can circle back and improve it in a subsequent iteration whenever someone has time to do a bit of research on vercel secrets.

**Review by cprice404 — COMMENTED** [review]
I think @NoSQLKnowHow makes a fair point about removing the Momento items from your list of prereqs. otherwise this looks like a great starting point to me and we should go ahead and get it merged so that we can release the blog post about it! We can continue to refine it after that.

**Review by cprice404 — APPROVED** [review]
@NoSQLKnowHow we have a meeting in a bit that this content is very relevant for so I'm approving so we can get it merged. Totally happy to make/accept further tweaks though!

**Pull request merged**